src/app/client/projects/desktop/src/app/modules/offline/components/content-manager/content-manager.component.ts

```diff
@@ -76,9 +80,21 @@ export class ContentManagerComponent implements OnInit, OnDestroy {
           this.contentResponse = _.filter(apiResponse, (o) => {
             return o.status !== 'canceled';
           });
+          this.getNoSpaceContentList(apiResponse);
         });
   }
-
+  getNoSpaceContentList(allContentList) {
+    this.noSpaceContentList = _.filter(allContentList, (content) =>
```

Comment:
noSpaceContentList not required. do forEach and to listToShow, if it's not in previously shown and not in listToShow also